Sources/FoundationXML/XMLParser.swift

```diff
@@ -575,9 +588,33 @@ open class XMLParser : NSObject {
 
         return result
     }
-    
+#endif
+
+    internal func parse(from data: Data) -> Bool {
+        var data = data
+        var result = true
+        let buffer = malloc(_chunkSize)!.bindMemory(to: UInt8.self, capacity: _chunkSize)
```

Comment:
Where is this used?

Reply:
It's used in `open func parse()` below.

Reply:
I mean the buffer allocated on this line. It doesn’t seem to be used AFAICT.